r1cs/gadgets/std/src/fields/nonnative/mod.rs

```diff
@@ -1,39 +1,32 @@
-/// This module contains the implementation of three structs:
-///
-/// - `NonNativeFieldParams` specifies the constraint prime field (called `ConstraintF`),
-///     the simulated prime field (called `SimulationF`), and internal parameters.
-/// - `NonNativeFieldGadget` implements the `FieldGadget` for simulating `SimulationF`
-///     arithmetic within `ConstraintF`.
-/// - `NonNativeFieldMulResultGadget` is an intermediate representations of the
-///     result of multiplication, which is hidden from the `FieldGadget` interface
-///     and is left for advanced users who want better performance.
-///
-/// DISCLAIMER: This is entirely a porting of [arkworks/nonnative](https://github.com/arkworks-rs/nonnative)
-///             to work with our GingerLib.
-
+//! A module for simulating non-native field arithmetics according to [Kosba et al.](https://ieeexplore.ieee.org/document/8418647),
+//! using some of the optimizations from [Ozdemir et al.](https://eprint.iacr.org/2019/1494).
+//! Ported from [arkworks/nonnative](https://github.com/arkworks-rs/nonnative).
+//! The following types are defined/supported:
+//!
```

Comment:
Cargo doc skips everything below the empty line 5. Let us remove it.

Reply:
Done